Oracle/encounter.sql

```diff
@@ -2,153 +2,173 @@
 */
 insert into cdm_status (task, start_time) select 'encounter', sysdate from dual
 /
+
```

Comment:
Encounter was completely refactored to address a complete breakdown in performance caused by a single additional join on observation_fact.  In the wake of the o_f join, encounter went from a 2 hour runtime to 15 hours.  Refactoring brought it back down to 1 hour.  

Med_admin ended up in a similar situation based on a encounter.  You'd think there would be an obvious lesson but other script have stood up to single joins without going off the rails.